scripts/appscript.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+set -eu -o pipefail
+
+[[ $# -lt 1 ]] && {
```

Comment:
because there are 5 required vars...
```suggestion
[[ $# -lt 5 ]] && {
```